single-line-diagram/single-line-diagram-core/src/main/java/com/powsybl/sld/svg/LabelProvider.java

```diff
@@ -76,6 +76,9 @@ public LabelPosition getPosition() {
 
     }
 
+    record NodeLegend(String label, String className, String idName) {
```

Comment:
You cannot speak of className here, as we splitted class / label responsibilities between `LabelProvider` and `StyleProvider`. But should we really make a difference between angle and voltage? Right now it's the same style which is applied for both: in both components.css
```css
.sld-angle, .sld-voltage {font: 10px serif}
```
We could take the opportunity to simplify by adding a single `sld-node-legend` style for each, or do you need specific styles for each line? For specific styles, we need to add a new `getBusInfoLegendStyle(String type)` method in `StyleProvider`, quite heavy for an unused feature.

Besides, `idName`, is it a name or an id? It's not an id as it's not unique (we'll soon have several legends in substation diagrams). So we'd better rename it into `name` or something like `type`

Lastly, you could enclose this into `ElectricalNodeInfo` record as it's only used there, and simply rename it to `Caption`. Indeed, naming it `legends` is a bit strange, I think there's a single legend (which is all the infos together), but several "captions" for each bus circle.

Reply:
replaced 
`.sld-angle, .sld-voltage {font: 10px serif}`

with
`.sld-node-legend {font: 10px serif}`